PendingReleaseNotes

```diff
@@ -164,6 +164,14 @@
   5 seconds. The feature is on by default. A new config option ``enable_availability_tracking``
   can be used to turn off the feature if required. Another command is added to clear the 
   availability status for a specific pool, ``ceph osd pool clear-availability-status <pool-name>``. 
+  users to view the availability score for each pool in a cluster. A pool is considered 
```

Comment:
Something seems amiss here, the previous (unchanged) line ends in a full stop. Then the added text starts with a lower case letter and seems to be just a fragment of a sentence?

Reply:
Sorry, I had missed that one and it totally makes sense with that change. Looks good then.

Reply:
@bluikko would it be possible for you to approve https://github.com/ceph/ceph/pull/65521?

Reply:
> @bluikko would it be possible for you to approve #65521?

Done, with the caveat listed in that review comment.